### DISCUSSION TITLE
Continuously train my custom running ner model with different kinds of datasets

### DISCUSSION THREAD

**Discussion by dhavalaprashanth** [explosion/spaCy#12581] [Help: Coding & Implementations]
Hi, I am completely new to spacy and training ml models. So my question is 

How can i keep my model running everytime and feed that model with my training dataset and completely generate a new model that  is learnt from my datasets and save it

**Comment by adrianeboyd** [comment]
Sorry, I'm not sure I understand what you're asking? The spacy course is a good place to start for an introduction to spacy: https://course.spacy.io
